src/pages/DiscountPage.js

```diff
@@ -0,0 +1,36 @@
+class DiscountPage {
+    oneColumn = "tr> td:nth-child(2)  > p"
+    twoColumn = "tr > td:nth-child(3)"
+    threeColumn = "tr > td:nth-child(4)"
+
+    getPrices(prices) {
+        let priceOneValue = 0
+        let priceTwoValue = 0
+        let priceResultValue = 0
+        cy.get(this.oneColumn).then(($col) => {
+            $col.each((index, el) => {
+                cy.wrap(el).invoke('text').then(text => {
+                    priceOneValue += parseFloat(text);
+                })
+            })
+        })
+        cy.get(this.twoColumn).then(($col) => {
+            $col.each((index, el) => {
+                cy.wrap(el).invoke('text').then(text => {
+                    priceTwoValue += parseFloat(text);
+                })
+            })
+        })
+        cy.get(this.threeColumn).then(($col) => {
+            $col.each((index, el) => {
+                cy.wrap(el).invoke('text').then(text => {
+                    priceResultValue += parseFloat(text);
+                })
+            })
+        })
+        cy.then(() => {
+            prices.push(priceOneValue, priceTwoValue, priceResultValue)
+        })
+    }
+}
+export default new DiscountPage();
```

Comment:
No need to create PO file.
You need to create a separate file with functions.
Function should be able to get any number of elements, and should has in it's body a single THEN block.